### PR TITLE
:arrow_up: bump caikit

### PR DESCRIPTION
**Pull request by joerunde**
Allows the latest caikit release

### CODE REVIEW

**Review by evaline-ju — APPROVED** [review]
LGTM

**Review by evaline-ju — APPROVED** [review]
LGTM

File: tests/test_tgis_backend.py [hunk 1]
```diff
@@ -65,10 +65,7 @@ def mock_tgis_proc_fail():
 
 def test_tgis_backend_is_registered():
     """Make sure that the TGIS backend is correctly registered with caikit"""
-    assert (
-        TGISBackend.backend_type
-        in caikit.core.module_backends.backend_types.MODULE_BACKEND_TYPES
-    )
+    assert hasattr(caikit.core.module_backends.backend_types, TGISBackend.backend_type)
```
Comment (joerunde) [inline tests/test_tgis_backend.py:68]: I'm like... reasonably certain that many prior versions of caikit supported this, based on the overridden `__getattr__` on `caikit.core.module_backends.backend_types`

**Review by tjohnson31415 — APPROVED** [review]

File: tests/test_tgis_backend.py (hunk 1 above)
Comment (gabe-l-hart) [reply tests/test_tgis_backend.py:68]: Yep, they did

**Review by gabe-l-hart — APPROVED** [review]

**Pull request merged**